test/alda/lisp/events/note_test.clj

```diff
@@ -0,0 +1,78 @@
+(ns alda.lisp.events.note-test
+  (:require [alda.lisp.events.note :as note]
+            [clojure.test :refer :all])
+  (:import [alda.lisp.model.records AbsoluteOffset]))
+
+(deftest event-updates-test
+  (testing "MIDI note inside range parsing correctly"
+    (let [score {:chord-mode false
```

Comment:
I worry that representing the value of the score map inline here might be coupling too closely to the current implementation. I think this should be expressed instead of in terms of behavior/actions.

In this case, I think we just want to initialize a score and add a piano to it, so we can represent that as `(score (part "piano"))`.